cmd/grill/main.go

```diff
@@ -94,5 +94,14 @@ func Main(a []string, stdout, stderr io.Writer) int {
 		}
 	}
 
+	if *opts.keepTmpdir {
+		_, err := fmt.Fprintf(stderr, "# Kept temporary directory: %s\n", context.WorkDir)
+		if err != nil {
+			log.Println(err)
+		}
```

Comment:
should panic on error, as log will try to write to stderr as well

Reply:
Makes sense; PTAL

Reply:
Oh, nevermind, I see you already approved 👍